### PR TITLE
Updates due to 2022 election

### PR DESCRIPTION
**Pull request by brownsarahm**
- begin language change agreement to renewal

### CODE REVIEW

**Comment by brownsarahm** [comment]
still need to:
- [ ] change chair/secretary after internal election
- [ ] discuss phrasing for the renewal document
- [ ] determine where the renewal doc will live

**Review by karenword — APPROVED** [review]

File: powers_responsibilities.md [hunk 1]
```diff
@@ -3,16 +3,16 @@ This document details the powers and responsibilities of [Instructor Trainers](h
...
 * Ratify initial governance structure?
 * Communicate with Trainers, Leadership committee & Core Team
 * Set agenda for Trainer meetings
 * Volunteer or nominate Trainers for elected and appointed roles
 
 ### What powers belong to Leadership?
-* Interpret Trainer Agreement, including assignment of units to activities not listed in the Agreement
-* Propose of changes to Trainer Agreement, including potential enforcement
+* Interpret Trainer Certification Renewal data
+* Propose of changes to Trainer Certification Renewal, including potential enforcement
```
Comment (karenword) [inline powers_responsibilities.md:15]:
```suggestion
* Propose changes to Trainer Certification Renewal process
```

Comment (karenword) [reply powers_responsibilities.md:15]: (Enforcement is no longer "potential" but part of the process)

**Comment by karenword** [comment]
I suggest merging this now to correct out-of-date information, then separately updating chair, secretary, phrasing related to certification renewal, and any changes to the location of certification renewal documentation (currently in handbook, in location where Trainer Agreement was previously)

**Review by karenword — APPROVED** [review]

**Comment by brownsarahm** [comment]
This pull request:
- restores the 2021 nominees 
- moves both nominee lists to a subfolder
- adds separate result files so that past members of the panel are findable without using git history
- changes language about trainer agreement to renewal process

**Review by karenword — APPROVED** [review]

**Pull request merged**